src/app/explore/page.module.css

```diff
@@ -1,77 +1,73 @@
-.pageHeader{
-    display: flex;
-    flex-direction: column-reverse;
-    justify-content: center;
-    align-items: center;
-    margin: 3rem 0 5rem 0;
-    color: #ffff;
+.pageHeader {
+	display: flex;
+	flex-direction: column-reverse;
+	justify-content: center;
+	align-items: center;
+	margin: 3rem 0 5rem 0;
+	color: #ffff;
```

Comment:
Why you change the spacing. Who said you to change it. Make it as it was earlier

Reply:
> Why you change the spacing. Who said you to change it. Make it as it was earlier

ok , that's my bad , i apoloize for that